dcap_quoteverify/types/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+
+//! Rust wrappers for SGX DCAP quoteverify types
+
+#![no_std]
+
+#[cfg(test)]
+#[macro_use]
+extern crate yare;
+
+use mc_sgx_dcap_quoteverify_sys_types::sgx_qv_path_type_t;
+
+#[derive(Eq, PartialEq, Debug)]
+pub enum Path {
+    QVE,
+    QPL,
+}
+
+impl From<sgx_qv_path_type_t> for Path {
+    fn from(p: sgx_qv_path_type_t) -> Self {
+        match p {
+            sgx_qv_path_type_t::SGX_QV_QVE_PATH => Self::QVE,
+            sgx_qv_path_type_t::SGX_QV_QPL_PATH => Self::QPL,
+            p => panic!("Unexpected path: {:?}", p),
```

Comment:
If this path is taken do we want a panic, or should we limit enum types to using [`TryFrom`](https://doc.rust-lang.org/std/convert/trait.TryFrom.html)

Reply:
It should just be `TryFrom`, yeah.

Reply:
Thoughts for where to house the error.  I'm guessing we should use one common error for converting enums
```rust
#[derive(Debug)]
pub enum Error {
    EnumMappingError(String),
}
```

Reply:
nvm restacked on #81 and will consolidate the error messages